drivers/wifi/esp32/Kconfig.esp32

```
@@ -35,6 +35,12 @@ config ESP32_WIFI_STA_AUTO_DHCPV4
 	help
 	  WiFi driver will automatically initiate DHCPV4 negotiation when connected.
 
+config ESP32_WIFI_APSTA_MODE
+	bool "Enable APSTA Mode"
```

```suggestion
	bool "Enable AP-STA concurrent Mode"
```
may be also document single channel or different channel?